reana/reana_dev/git.py

```diff
@@ -820,4 +874,45 @@ def _push_to_origin(components):
         _push_to_origin(components)
 
 
+@click.option(
+    "--component",
+    "-c",
+    required=True,
+    multiple=True,
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@click.option(
+    "--next-version", "-v", help="Shall we manually specify component's next version?",
+)
+@git_commands.command(name="git-create-release-commit")
+def git_create_release_commit_command(component, next_version):  # noqa: D301
+    """Create a release commit for the specified components.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'DEMO' that will expand
+                               to include several runable REANA demo examples;
+                         * (7) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :param next_version: Manually specifies the version for the component. If not
+        provided the next version is autodiscovered.
```

Comment:
Instead of "autodiscovered", it would be more precise to say "If not provided, the last version will be auto-incremented."